keyboards/mt84/config.h

```diff
@@ -0,0 +1,107 @@
+ /* Copyright 2020  MT <704340378@qq.com>
+  * 
+  * This program is free software: you can redistribute it and/or modify 
+  * it under the terms of the GNU General Public License as published by 
+  * the Free Software Foundation, either version 2 of the License, or 
+  * (at your option) any later version. 
+  * 
+  * This program is distributed in the hope that it will be useful, 
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of 
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+  * GNU General Public License for more details. 
+  * 
+  * You should have received a copy of the GNU General Public License 
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+  */
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4D54  // "MT"
+#define PRODUCT_ID      0x0002
+#define DESCRIPTION     mt84
```

Comment:
`DEVICE_VER` is required to get the firmware to compile. `MANUFACTURER` and `PRODUCT` assist with device identification for users.

`DESCRIPTION` is deprecated and not used anywhere in QMK anymore.

```suggestion
#define DEVICE_VER      0x0001
#define MANUFACTURER    MT
#define PRODUCT         MT84
```